server_admin/topics/sessions/timeouts.adoc

```diff
@@ -13,38 +13,38 @@ Let's walk through each of the items on this page.
 |===
 |Configuration|Description
 
+|Default Signature Algorithm
+|The default algorithm that is used to assign tokens for this realm.
+
 |Revoke Refresh Token
-|For OIDC clients that are doing the refresh token flow, this flag, if on, will revoke that refresh token and issue another with the request that the client has to use.
- This basically means that refresh tokens have a one time use.
+|For OIDC clients that are doing the refresh token flow, this flag, if on, will revoke that refresh token and issue another token with the request that the client has to use. The result is that each refresh token is used only once. 
 
 |SSO Session Idle
-|Also pertains to OIDC clients.  If the user is not active for longer than this timeout, the user session will be invalidated.  How is idle time checked?
-A client requesting authentication will bump the idle timeout.  Refresh token requests will also bump the idle timeout.
-There is a small window of time that is always added to the idle timeout before the session is actually invalidated (See note below).
+|Also pertains to OIDC clients.  If the user is not active for longer than this timeout, the user session will be invalidated.  The idle timeout is reset by a client requesting authentication or by a refresh token request.
+There is a small window of time that is always added to the idle timeout before the session  invalidation takes effect. See the note below.
 
 |SSO Session Max
-|Maximum time before a user session is expired and invalidated.  This is a hard number and time.  It controls the maximum time
- a user session can remain active, regardless of activity.
+|Maximum time before a user session is expired and invalidated. This option controls the maximum time that a user session can remain active, regardless of activity.
```

Comment:
"This option controls the maximum time that a user session can remain active, regardless of activity."

This seems a bit confusing because "active" / "activity" is used both in relation to the user session and the user.

I suggest one of the following alternatives:
- "This option controls the maximum time that a user session can remain active, regardless of **user** activity."
- "This option controls the maximum time that a user session can remain **open,** regardless of **user** activity."
- "This option controls the maximum time that a user session can remain **valid,** regardless of **user** activity."

Also not sure if the word "active" should be used to describe an offline token (few cells below in the same table).

RFC @mposolda @hmlnarik WDYT?

Reply:
I would like someone from the dev team to verify the above suggestion before applying changes.

Reply:
+1 for @tkyjovsk suggestions. I like probably the very first sentence most, but that's just me. This one:
"This option controls the maximum time that a user session can remain active, regardless of **user** activity."

Reply:
@mposolda Ok, thanks. What about the two offline token options a few lines below?

"...  maximum time the offline token can remain active, regardless of activity."

Same?